<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tz40widget</class>
 <widget class="QWidget" name="tz40widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1163</width>
    <height>617</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page">
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>-110</y>
         <width>1071</width>
         <height>771</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="resources.qrc">:/Resources/fourties.jpg</pixmap>
       </property>
      </widget>
      <widget class="QPushButton" name="jointButton">
       <property name="geometry">
        <rect>
         <x>610</x>
         <y>280</y>
         <width>41</width>
         <height>91</height>
        </rect>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">background-image: url(:/Resources/joint.jpg);
background-repeat: no-repeat;
background-color: rgb(255, 255, 255);
border:none;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QFrame" name="gameFrame">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>0</y>
         <width>711</width>
         <height>541</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(85, 85, 0);</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QSlider" name="slider4">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>400</y>
          <width>561</width>
          <height>20</height>
         </rect>
        </property>
        <property name="maximum">
         <number>2</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
       <widget class="Line" name="line">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>20</y>
          <width>20</width>
          <height>471</height>
         </rect>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>290</x>
          <y>20</y>
          <width>91</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">border-width: 1px;
border-style: solid;
border-color: rgb(100, 100, 0);</string>
        </property>
        <property name="text">
         <string>1940s</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>460</x>
          <y>20</y>
          <width>161</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">border-width: 1px;
border-style: solid;
border-color: rgb(100, 100, 0);</string>
        </property>
        <property name="text">
         <string>Post 1940s</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QSlider" name="slider3">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>310</y>
          <width>561</width>
          <height>20</height>
         </rect>
        </property>
        <property name="maximum">
         <number>2</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
       <widget class="QSlider" name="slider2">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>220</y>
          <width>561</width>
          <height>20</height>
         </rect>
        </property>
        <property name="maximum">
         <number>2</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
       <widget class="QSlider" name="slider1">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>130</y>
          <width>561</width>
          <height>20</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="maximum">
         <number>2</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="tickPosition">
         <enum>QSlider::NoTicks</enum>
        </property>
       </widget>
       <widget class="QSlider" name="slider5">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>490</y>
          <width>561</width>
          <height>20</height>
         </rect>
        </property>
        <property name="maximum">
         <number>2</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
       <widget class="Line" name="line_2">
        <property name="geometry">
         <rect>
          <x>420</x>
          <y>20</y>
          <width>20</width>
          <height>471</height>
         </rect>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>20</y>
          <width>161</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">border-width: 1px;
border-style: solid;
border-color: rgb(100, 100, 0);</string>
        </property>
        <property name="text">
         <string>Pre 1940s</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QPushButton" name="submitButton">
        <property name="geometry">
         <rect>
          <x>530</x>
          <y>510</y>
          <width>80</width>
          <height>25</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(108, 108, 0);</string>
        </property>
        <property name="text">
         <string>Submit</string>
        </property>
       </widget>
       <widget class="QLabel" name="slinky">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>60</y>
          <width>68</width>
          <height>68</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 85, 127);</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="resources.qrc">:/Resources/slinky.jpg</pixmap>
        </property>
       </widget>
       <widget class="QLabel" name="bandAid">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>150</y>
          <width>68</width>
          <height>68</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 85, 127);</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="resources.qrc">:/Resources/band-aid.jpg</pixmap>
        </property>
       </widget>
       <widget class="QLabel" name="jeep">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>240</y>
          <width>68</width>
          <height>68</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 85, 127);</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="resources.qrc">:/Resources/jeep.jpg</pixmap>
        </property>
       </widget>
       <widget class="QLabel" name="barbie">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>330</y>
          <width>68</width>
          <height>68</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 85, 127);</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="resources.qrc">:/Resources/barbie.jpg</pixmap>
        </property>
       </widget>
       <widget class="QLabel" name="microwave">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>420</y>
          <width>68</width>
          <height>68</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 85, 127);</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="resources.qrc">:/Resources/microwave.jpg</pixmap>
        </property>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>620</x>
          <y>90</y>
          <width>55</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Slinky</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>620</x>
          <y>180</y>
          <width>55</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Band-Aid</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>630</x>
          <y>270</y>
          <width>55</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Jeep</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_8">
        <property name="geometry">
         <rect>
          <x>630</x>
          <y>360</y>
          <width>55</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Barbie</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_9">
        <property name="geometry">
         <rect>
          <x>620</x>
          <y>450</y>
          <width>71</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Microwave</string>
        </property>
       </widget>
       <widget class="QLabel" name="incorrectWarning">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>510</y>
          <width>321</width>
          <height>20</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(54, 54, 0);
padding:2px;</string>
        </property>
        <property name="text">
         <string>One or more answers are incorrect. Please try again</string>
        </property>
       </widget>
       <widget class="QPushButton" name="closeButton">
        <property name="geometry">
         <rect>
          <x>669</x>
          <y>10</y>
          <width>31</width>
          <height>25</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(85, 85, 0);
border-color: rgb(77, 77, 0);</string>
        </property>
        <property name="text">
         <string>X</string>
        </property>
       </widget>
       <zorder>line_2</zorder>
       <zorder>line</zorder>
       <zorder>slider4</zorder>
       <zorder>label_2</zorder>
       <zorder>label_3</zorder>
       <zorder>slider3</zorder>
       <zorder>slider2</zorder>
       <zorder>slider1</zorder>
       <zorder>slider5</zorder>
       <zorder>label_4</zorder>
       <zorder>submitButton</zorder>
       <zorder>slinky</zorder>
       <zorder>bandAid</zorder>
       <zorder>jeep</zorder>
       <zorder>barbie</zorder>
       <zorder>microwave</zorder>
       <zorder>label_5</zorder>
       <zorder>label_6</zorder>
       <zorder>label_7</zorder>
       <zorder>label_8</zorder>
       <zorder>label_9</zorder>
       <zorder>incorrectWarning</zorder>
       <zorder>closeButton</zorder>
      </widget>
      <widget class="QPushButton" name="startButton">
       <property name="geometry">
        <rect>
         <x>480</x>
         <y>300</y>
         <width>101</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(85, 85, 0);</string>
       </property>
       <property name="text">
        <string>Start</string>
       </property>
      </widget>
      <widget class="QPushButton" name="tokenButton">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>290</y>
         <width>121</width>
         <height>121</height>
        </rect>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">border-style: solid;
background-image: url(:/Resources/token.png);
 border-width:1px;
 border-radius:50px;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="buttonImage" stdset="0">
        <pixmap resource="resources.qrc">:/Resources/token.png</pixmap>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="page_2"/>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
